lldb/include/lldb/Target/RegisterFlags.h

```diff
@@ -9,10 +9,16 @@
 #ifndef LLDB_TARGET_REGISTERFLAGS_H
 #define LLDB_TARGET_REGISTERFLAGS_H
 
-#include "lldb/Utility/Log.h"
+#include <cassert>
```

Comment:
Not needed in this header, move to .cpp if needed there?

Reply:
Moved the assert into the cpp file and removed the include in the header.